src/GraphQL/Utilities/Visitors/BaseSchemaNodeVisitor.cs

```diff
@@ -34,21 +34,32 @@ public virtual void VisitInputObject(IInputObjectGraphType type, ISchema schema)
         }
 
         /// <inheritdoc />
-        public virtual void VisitFieldDefinition(FieldType field, ISchema schema)
+        public virtual void VisitFieldDefinition(FieldType field, IObjectGraphType type, ISchema schema)
```

Comment:
Since we have added new methods, I propose to unify their names:
```csharp
public virtual void VisitObjectFieldDefinition(FieldType field, IObjectGraphType type, ISchema schema);
public virtual void VisitInterfaceFieldDefinition(FieldType field, IInterfaceGraphType type, ISchema schema);
public virtual void VisitInputObjectFieldDefinition(FieldType field, IInputObjectGraphType type, ISchema schema);
public virtual void VisitObjectFieldArgumentDefinition(QueryArgument argument, FieldType field, IObjectGraphType objectGraphType, ISchema schema);
public virtual void VisitInterfaceFieldArgumentDefinition(QueryArgument argument, FieldType field, IInterfaceGraphType objectGraphType, ISchema schema);
```

There will be no confusion by giving them different names.

Reply:
#2340